### PR TITLE
Add periodic fastforward

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

File: ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: open-cluster-management
+    repo: registration-operator
+  name: periodic-open-cluster-management-registration-operator-fast-forward
+  spec:
+    containers:
+    - args:
+      - --config-dir=./ci-operator/config/open-cluster-management/registration-operator/
```
Comment (droslean) [inline ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml:14]: There is no such dir in `open-cluster-management/registration-operator`

File: ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      - --current-release=master
+      - --future-release=2.1
+      - --fast-forward
+      - --confirm
+      - --username=openshift-merge-robot
```
Comment (droslean) [inline ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml:19]: It seems also that you are using different token. It doesn't make sense to use `openshift-merge-robot` as the username.

File: ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml (hunk 1 above)
Comment (droslean) [reply ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml:14]: You will need to add the `openshift/release@master` in the extra refs. Also, I don't think that you need the existing extra_refs you have now.

Comment (qiujian16) [reply ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml:14]: i want to fast-forward on https://github.com/open-cluster-management/registration-operator, not openshift actually. Will I be able to do that?

Comment (droslean) [reply ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml:14]: The `./ci-operator/config/open-cluster-management/registration-operator/` config exists in `openshift/release` repo. So you need to clone that one instead.

Comment (qiujian16) [reply ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml:14]: ah, i see. Updated. How could this job know which repo to fast-forward?  By reading zz_generated_metadata in config yaml?

Comment (droslean) [reply ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml:14]: The tool will read all the configuration files that exist in the specified `--config-dir`

File: ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml (hunk 2 above)
Comment (qiujian16) [reply ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml]: updated.

**Comment by qiujian16** [comment]
@droslean would you help to review a look again

**Pull request closed without merging**